indexing-service/src/main/java/org/apache/druid/indexing/common/task/HadoopIndexTask.java

```diff
@@ -202,7 +202,7 @@ public String getType()
   @Override
   public Set<ResourceAction> getInputSourceResources()
   {
-    return Collections.singleton(new ResourceAction(new Resource(ResourceType.EXTERNAL, "hadoop"), Action.READ));
+    return Collections.singleton(new ResourceAction(new Resource( "hadoop", ResourceType.EXTERNAL), Action.READ));
```

Comment:
Nit: there's an extra whitespace:
```suggestion
    return Collections.singleton(new ResourceAction(new Resource("hadoop", ResourceType.EXTERNAL), Action.READ));
```

Reply:
fixed